src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -271,69 +201,24 @@ private static async Task WriteLogFile(ZipArchive archive, CancellationToken tok
 
         private async Task<Stream> GetDebugInfoFromNodeAsync(
             JsonOperationContext context,
-            RequestExecutor requestExecutor,
-            IEnumerable<string> databaseNames)
+            RequestExecutor requestExecutor)
         {
-            var bodyJson = new DynamicJsonValue
-            {
-                [nameof(NodeDebugInfoRequestHeader.FromUrl)] = ServerStore.GetNodeHttpServerUrl(),
-                [nameof(NodeDebugInfoRequestHeader.DatabaseNames)] = databaseNames
-            };
-
             using (var ms = new MemoryStream())
             using (var writer = new BlittableJsonTextWriter(context, ms))
             {
-                context.Write(writer, bodyJson);
+                context.Write(writer, new DynamicJsonValue());
```

Comment:
what is that?

Reply:
bodyJson originally contained 2 parameters for the endpoint that is called in the next line. Now it has been replaced with a different endpoint so the params aren't needed anymore (and the Write func still requires a 2nd argument)

Reply:
removed